runtimes/eden/Cargo.toml

```diff
@@ -177,7 +177,10 @@ pallet-transaction-payment-rpc-runtime-api = { git = "https://github.com/parityt
 pallet-uniques = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.29" }
 pallet-utility = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.29" }
 pallet-aura = { git = "https://github.com/paritytech/substrate.git", default-features = false, branch = "polkadot-v0.9.29" }
-#pallet-xcm = { git = "https://github.com/paritytech/polkadot.git", default-features = false, branch = "release-v0.9.29" }
+
+cumulus-pallet-xcmp-queue = { git = "https://github.com/paritytech/cumulus", branch = "polkadot-v0.9.29", default-features = false }
+pallet-xcm = { git = "https://github.com/paritytech/polkadot.git", default-features = false, branch = "release-v0.9.29" }
```

Comment:
Please add these two pallets to `list_benchmark` and `add_benchmark` statements in runtimes/eden/src/lib.rs and test the result of benchmarking for them locally. @simonsso can help you with the instruction for benchmarking and check if your result is what it should be.

Reply:
https://github.com/NodleCode/chain-workspace/issues/259 is going to address benchmarking. So I mark this as resolved.